api/src/routing.js

```diff
@@ -299,10 +299,8 @@ app.postJson('/api/sms', function(req, res) {
     .catch(err => serverUtils.error(err, req, res));
 });
 
-app.all('/api/v1/export/:type/:form?', exportData.routeV1);
```

Comment:
My policy is not to remove them unless it's impractical to maintain them. In this case it's not possible to keep the xml type and have streaming. As feedback is the last type that still worked at this route it made sense to me to remove it. I'm fairly sure nobody is using this API so I think it's ok to drop it just like we did for the other export types. I'll document it in the release notes just in case.